CHANGELOG.md

```diff
@@ -7,6 +7,9 @@
 * Add support of c++ files (`hpp` and `cpp`) in `PBXFileReference:set_last_known_file_type`.  
   [romanmikhailov](https://github.com/romanmikhailov)
   [#783](https://github.com/CocoaPods/Xcodeproj/issues/783)
+* Add support of dependency analysis option in `PBXShellScriptBuildPhase:always_out_of_date`.
```

Comment:
nit: add one line here.

Also add two empty spaces at the end of this line for markdown formatting.